{
    "id": "CVE-2016-1902",
    "modified": "2018-05-23T12:56:38Z",
    "published": "2016-01-25T20:10:39Z",
    "aliases": [],
    "related": [],
    "summary": "CVE-2016-1902: SecureRandom's fallback not secure when OpenSSL fails ",
    "details": "",
    "affected": {
        "package": {
            "ecosystem": "Packagist",
            "name": "symfony\/security",
            "purl": "pkg:packagist\/symfony\/security"
        },
        "ranges": {
            "type": "SEMVER",
            "events": [
                {
                    "introduced": ">=2.3.0"
                },
                {
                    "fixed": "<2.3.37"
                },
                {
                    "introduced": ">=2.4.0"
                },
                {
                    "fixed": "<2.5.0"
                },
                {
                    "introduced": ">=2.5.0"
                },
                {
                    "fixed": "<2.6.0"
                },
                {
                    "introduced": ">=2.6.0"
                },
                {
                    "fixed": "<2.6.13"
                },
                {
                    "introduced": ">=2.7.0"
                },
                {
                    "fixed": "<2.7.9"
                }
            ]
        }
    },
    "references": [
        {
            "type": "PACKAGE",
            "url": "https:\/\/packagist.org\/packages\/symfony\/security"
        },
        {
            "type": "ADVISORY",
            "url": "https:\/\/symfony.com\/cve-2016-1902"
        }
    ]
}